src/commands/account/get_public_key/from_keychain/mod.rs

```diff
@@ -0,0 +1,114 @@
+use color_eyre::eyre::WrapErr;
+
+use crate::common::JsonRpcClientExt;
+use crate::common::RpcQueryResponseExt;
+
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(input_context = crate::GlobalContext)]
+#[interactive_clap(output_context = PublicKeyFromKeychainContext)]
+pub struct PublicKeyFromKeychain {
```

Comment:
It's a bit weird, to be honest. Maybe we shouldn't have that. Because to get a key from a keychain, we need to fetch a key before-hand (e.g., from the chain). So it doesn't really make sense.
Maybe we can iterate somehow over keychain entries and parse it that way....

Reply:
https://github.com/user-attachments/assets/34f42034-0f71-47f7-8906-41000eaca449